src/ff7_opengl.cpp

```diff
@@ -291,6 +291,11 @@ void ff7_init_hooks(struct game_obj *_game_object)
 		patch_code_int(ff7_externals.field_init_viewport_values + 0x6E, 240);
 	}
 
+	// ########################
+	// field direct color black
+	// ########################
+	patch_code_short(0x63F325 + 0xE3, 0x421);
```

Comment:
Currently my patch replace the faulty color by grey, this is not enough to get the original rendering

![2023-08-30 23_49_35-Final Fantasy VII (Direct3D11) devel](https://github.com/julianxhokaxhiu/FFNx/assets/8015696/4ae7c422-f4b3-4b40-94d7-6a042d365c1c)

I tried set this value to 0, but in this case the pixel is transparent, not black, so you will have artifacts between bg layers

Reply:
A darker blue works well for this field
`patch_code_short(0x63F325 + 0xE3, 0x1);`
![2023-08-31 13_20_25-Final Fantasy VII (Direct3D11) devel](https://github.com/julianxhokaxhiu/FFNx/assets/8015696/f9c19250-29a5-45b7-a84f-aa2db382170a)